README.md

```diff
@@ -4,41 +4,41 @@
 
 [<img src="https://github.com/MaterializeInc/materialize/assets/23521087/39270ecb-7ac4-4829-b98b-c5b5699a16b8" width=35%>](https://materialize.com)
 
-Materialize is the real-time data integration platform that enables you to use SQL to transform, deliver, and act on fast changing data.
+Materialize is a real-time data integration platform that creates and continually updates consistent views of transactional data from across your organization. Its SQL interface democratizes the ability to serve and access live data. Materialize can be deployed anywhere your infrastructure runs.
+
+Use Materialize to do things like deliver fresh context for AI/RAG pipelines, power operational dashboards, and create more dynamic customer experiences without building time-consuming custom data pipelines.
+
+
+The three most common patterns for adopting Materialize are the following:
+
+- Query Offload (CQRS) - Scale complex read queries more efficiently than a read replica, and without the headaches of cache invalidation.
+- Integration Hub (ODS) - Extract, load, and incrementally transform data from multiple sources. Create live views of your data that can be queried directly or pushed downstream.
+- Operational Data Mesh (ODM) - Use SQL to create and deliver real-time, strongly consistent data products to streamline coordination across services and domains.
 
-Using SQL and common tools in the wider data ecosystem, Materialize allows you
-to build real-time automation, engaging customer experiences, and interactive
-data products that drive value for your business while reducing the cost of
-[data freshness](https://materialize.com/blog/decouple-cost-and-freshness/).
 
 ## Get started
 
-Ready to try out Materialize? [Sign up](https://materialize.com/register/) to get started! 🚀
+Ready to try out Materialize? You can [sign up](https://materialize.com/register/) for a free cloud trial or [download](https://materialize.com/download/) our community edition, which is free forever for deployments using less than 24 GiB of memory and 48 GiB of disk!
```

Comment:
~~Should we make it "the self-managed community edition" ? or somewhere stick in self-managed?~~
Oh ... I see how we're using enterprise edition and community edition below.